capplets/time-admin/src/time-share.c

```diff
@@ -0,0 +1,154 @@
+*         @Msg             Message content           
+*         @nType           Message type
+* Output: 
+*        
+* Author:  zhuyaliang  25/05/2018
+******************************************************************************/
+int MessageReport(const char *Title,const char *Msg,int nType)
+{
+    GtkWidget *dialog = NULL;
+    int nRet;
+
+    switch(nType)
+    {
+        case ERROR:
+        {
+            dialog = gtk_message_dialog_new(GTK_WINDOW(WindowLogin),
+                                            GTK_DIALOG_DESTROY_WITH_PARENT,
+                                            GTK_MESSAGE_ERROR,
+                                            GTK_BUTTONS_OK,
+                                            "%s",Title);
+            break;
+        }
+        case WARING:
+        {
+            dialog = gtk_message_dialog_new(GTK_WINDOW(WindowLogin),
+                                            GTK_DIALOG_DESTROY_WITH_PARENT,
+                                            GTK_MESSAGE_WARNING,
+                                            GTK_BUTTONS_OK,
+                                            "%s",Title);
+            break;
+        }
+        case INFOR:
+        {
+            dialog = gtk_message_dialog_new(GTK_WINDOW(WindowLogin),
+                                            GTK_DIALOG_DESTROY_WITH_PARENT,
+                                            GTK_MESSAGE_INFO,
+                                            GTK_BUTTONS_OK,
+                                            "%s",Title);
+            break;
+        }
+        case QUESTION:
+        {
+            dialog = gtk_message_dialog_new(GTK_WINDOW(WindowLogin),
+                                            GTK_DIALOG_DESTROY_WITH_PARENT,
+                                            GTK_MESSAGE_QUESTION,
+                                            GTK_BUTTONS_YES_NO,
+                                            "%s",Title);
+            gtk_dialog_add_button (GTK_DIALOG (dialog),("_Return"),
+                                   GTK_RESPONSE_ACCEPT);
+            break;
+        }
+        case QUESTIONNORMAL:
+        {
+            dialog = gtk_message_dialog_new(GTK_WINDOW(WindowLogin),
+                                            GTK_DIALOG_DESTROY_WITH_PARENT,
+                                            GTK_MESSAGE_QUESTION,
+                                            GTK_BUTTONS_YES_NO,
+                                            "%s",Title);
+            break;
+        }    
+        default :
+            break;
+
+    }
+    gtk_message_dialog_format_secondary_markup(GTK_MESSAGE_DIALOG(dialog),
+                                               TYPEMSG,
+                                               Msg);
+    gtk_window_set_title(GTK_WINDOW(dialog),("Message"));
+    nRet =  gtk_dialog_run(GTK_DIALOG(dialog));
+    gtk_widget_destroy(dialog);
+
+    return nRet;
+}
+void SetLableFontType(GtkWidget *Lable ,const char *Color,int FontSzie,const char *Word)
```

Comment:
@raveit65 
GtkInspector: `<span foreground='gray'weight='light'font_desc='11'>Time Zone:</span>`
```shell
$ git grep SetLableFontType
src/main.c:    SetLableFontType(TimeZoneLabel,"gray",11,_("Time Zone:"));
src/main.c:    SetLableFontType(NtpSyncLabel,"gray",11,_("Ntp Sync:"));
src/main.c:    SetLableFontType(TimeLabel,"green",13,_("Set Time"));
src/main.c:    SetLableFontType(DateLabel,"green",12,_("Set Date"));
```

Reply:
The color settings have been removed
```
SetLableFontType(DateLabel,13,_("Set Date"))
```